tests/test_compute_everything.py

```diff
@@ -35,8 +35,8 @@ def _compare_against_master(
                 np.testing.assert_allclose(
                     actual=data[p][name],
                     desired=master_data[p][name],
-                    atol=1e-10,
-                    rtol=1e-10,
+                    atol=1e-9,
```

Comment:
maybe try making atol to be `1e-12 * np.mean(np.abs(data[p][name]))` and then we can keep rtol tighter

Reply:
Ok I changed, but if this fails on CI due to floating point errors, it's your job to reduce to whatever makes is pass on your mac

Reply:
this also probably won't work on a different machine because if the latter is small, then atol gets smaller. small implies close to zero, so in that regiem rtol is not helpful

Reply:
fair point, I made a change that I think fixes this